pkg/model/test.go

```diff
@@ -52,6 +53,9 @@ func (test ManifestTests) Validate() error {
 	for _, t := range test {
 		if t != nil {
 			hasAtLeastOne = true
+			if len(t.Commands) == 0 {
+				return ErrEmptyCommands
+			}
```

Comment:
I think this would only check that the first non-nil test has commands, there is a break in line 59 that would prevent the rest of tests.

For example, with the following manifest, I'm not getting any error:

```
test:
  unit:
    commands:
      - echo "hola"
  integration:
    image: alpine
```